docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,337 @@
+
+This is where the React-Redux library comes in. It gives us a way to read data from the store in any component in our application, without needing
+to import the store into every file.
+
+To do this, we need to pass the store into a React-Redux `<Provider>` component.
+
+<Note type="success">
+
+#### Key Concept: `<Provider>`
+
+The React-Redux library has a `<Provider>` component. It uses [React's context API](https://reactjs.org/docs/context.html) to pass the Redux store
+to all components inside of itself.
+
+Every React-Redux app needs to put a `<Provider>` component around the main `<App>` component so that the whole app can use the store.
+
+</Note>
+
+We'll import the `Provider` component type from React-Redux, render a `<Provider>` around our `<App>`, and then pass the store to the `<Provider>`. (We'll also remove the console log, since we don't need that any more.)
+
+```js {4,18-25}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+import { Provider } from 'react-redux'
+
+import App from './App'
+
+function rootReducer(state, action) {
+  return 'Hello Redux!'
+}
+
+const store = configureStore({
+  reducer: rootReducer
+})
+
+const rootElement = document.getElementById('root')
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  rootElement
+)
+```
+
+### Show the State in the App Component
+
+The last step for the initial setup is to update our `<App>` component to read the state value from the Redux store.
+
+We've already seen that the React-Redux `<Provider>` component makes the store available to any components inside. To go along with that, React-Redux
+includes a React hook called `useSelector`, which React function components can call to retrieve the state values from the store.
+
+<Note type="success">
+
+#### Key Concept: "Selectors"
+
+A **_selector_** is a function that takes the Redux store state as an argument, and returns some part of that store state.
+
+The React-Redux `useSelector()` hook takes a selector function, calls the selector by passing in the store state, and then returns
+the selector function's return value to your component.
+
+You can also use selector functions elsewhere in your Redux code as well.
+
+</Note>
+
+Open up `App.js`, and rewrite it to look like this:
+
+```js {2,5-7,10,14}
+import React from 'react'
+import { useSelector } from 'react-redux'
+import './styles.css'
+
+function selectHelloText(state) {
+  return state
+}
+
+export default function App() {
+  const helloText = useSelector(selectHelloText)
+
+  return (
+    <div className="App">
+      <h1>{helloText}</h1>
+    </div>
+  )
+}
+```
+
+We import the `useSelector` function from React-Redux, and then create a function called `selectHelloText`. It receives whatever state is in the Redux
+store as an argument. For now, we'll just return that entire state value.
```

Comment:
I've seen beginners frequently return the entire state when they don't need to in `mapStateToProps`, causing unnecessary rerenders and defeating the purpose of this API. I understand that. There's just one value in the store in this case, but I think we should at least note that only the state the component needs should be returned, as Redux optimizes rerenders based on the result.